src/current/v24.2/show-external-connection.md

```diff
@@ -0,0 +1,70 @@
+---
+title: SHOW EXTERNAL CONNECTION
+summary: The SHOW EXTERNAL CONNECTION statement lists external connections.
+toc: true
+---
+
+{% include_cached new-in.html version="v24.2" %} You can use external connections to specify and interact with resources that are external from CockroachDB. When creating an external connection, you define a name for an external connection while passing the provider URI and query parameters. `SHOW EXTERNAL CONNECTION` displays the connection name, URI, and the type of connection.
+
+You can also use the following SQL statements to work with external connections:
+
+- [`CREATE EXTERNAL CONNECTION`]({% link {{ page.version.version }}/create-external-connection.md %})
+- [`SHOW CREATE EXTERNAL CONNECTION`]({% link {{ page.version.version }}/show-create-external-connection.md %})
+- [`DROP EXTERNAL CONNECTION`]({% link {{ page.version.version }}/drop-external-connection.md %})
+
+## Required privileges
+
+Users with the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) can view all external connections with `SHOW EXTERNAL CONNECTION`.
+
+Without the `admin` role, users can view the external connections that they own. Users own external connections that they have created with `CREATE EXTERNAL CONNECTION`.
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/show_external_connections.html %}
+</div>
+
+### Parameters
+
+`string_or_placeholder` is the name of the created external connection.
+
+## Responses
+
+Response | Description
+---------+------------
+`connection_name` | The user-specified name of the external connection.
+`connection_uri` | The [storage/sink]({% link {{ page.version.version }}/create-external-connection.md %}#supported-external-storage-and-sinks) URI that the external connection references.
+`connection_type` | `STORAGE` applies to [storage for backups and imports]({% link {{ page.version.version }}/use-cloud-storage.md %}) and [changefeed sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}).
```

Comment:
Maybe it's worth including the other types here? They are "KMS" and "FOREIGNDATA". See https://github.com/cockroachdb/cockroach/blob/f5beceda73bc2c04bad3e164613d6ef03deb2841/pkg/cloud/externalconn/connectionpb/connection.proto#L43

Reply:
Ah! thank you for grabbing these. I checked in with Alicia and confirmed we should document the postresql `foreigndata` connection type for pcr.